pkg/apis/config/v1alpha1/multi_cluster_service.go

```diff
@@ -39,6 +39,12 @@ type ClusterSpec struct {
 
 	// Name defines the name of the remote cluster.
 	Name string `json:"name,omitempty"`
+
+	// Weight defines the load balancing weight of the remote cluster
+	Weight int `json:"weight,omitempty"`
+
+	// Certificate defines the mTLS certificate of the remote cluster
+	Certificate string `json:"certificate,omitempty"`
```

Comment:
This PR does not mention anything regarding certificates. Please drop this change or describe in the commit why this is required.

Reply:
Certificate field was originally in the CRD, but not in the Go struct. I think this was a mistake for leaving it out. Thus adding it back in this PR.

Reply:
Where is the certificate being used? Is including it in the CRD necessary? Like I mentioned already, the commit message does not talk about the certificate, so seeing this here is quite confusing.

Reply:
I haven't checked if the `certificate` is used anywhere in the code base. If we find it's not needed, we can remove it in another PR.

Reply:
If you aren't sure it's needed, there's no reason to include it in this change.

Reply:
the crd is slate, the current crd doesn't have `certificate`. Please rebase changes and share a proposal as to why weight needs to be added for the remote cluster